spec/lib/hubspot/blog_spec.rb

```diff
@@ -1,116 +1,150 @@
 require 'timecop'
 
 describe Hubspot do
-  let(:example_blog_hash) do
-    VCR.use_cassette("blog_list", record: :none) do
-      url = Hubspot::Connection.send(:generate_url, Hubspot::Blog::BLOG_LIST_PATH)
-      resp = HTTParty.get(url, format: :json)
-      resp.parsed_response["objects"].first
-    end
-  end
-  let(:created_range_params) { { created__gt: false, created__range: (Time.now..Time.now + 2.years)  } }
-
   before do
     Hubspot.configure(hapikey: "demo")
-    Timecop.freeze(Time.local(2012, 'Oct', 10))
+    Timecop.freeze(Time.utc(2012, 'Oct', 10))
```

Comment:
This is the heart of the reason a few tests were failing on Travis. Changing this to utc will ensure the time zone is consistent across local and CI environments.